src/components/Surroundings.vue

```diff
@@ -110,12 +110,6 @@ export default {
   width: 110px;
 }
 
-.wrapper {
-  height: 100%;
```

Comment:
The wrapper no longer exists. We added it because otherwise when you hid the surroundings-output you lost the background. now we are just hiding all the child elements, so the background remains.

Reply:
I thought we had added the `height: 100%` to make it fill the pane when there were no children showing. That's good, I'll merge then.